compiler/src/dotty/tools/dotc/typer/CheckCaptures.scala

```diff
@@ -124,6 +124,7 @@ class CheckCaptures extends RefineTypes:
               val isGlobal = ref match
                 case ref: TypeRef => ref.isRootCapability
                 case ref: TermRef => ref.prefix != NoPrefix && ref.symbol.hasAnnotation(defn.AbilityAnnot)
+                case ref: TermParamRef => false
```

Comment:
I suggest
```suggestion
                case _ => false
```
TypeParamRefs and ThisTypes are also members of capture sets